src/pal/src/map/map.cpp

```diff
@@ -2138,6 +2138,27 @@ MAPRecordMapping(
     return palError;
 }
 
+static off_t
+MAPcalcAdj(
```

Comment:
Without looking into the function's code, it is not clear what it calculates. A brief comment (like near the neighbor functions) would be good.